proxy/common_neon/erc20_wrapper.py

```diff
@@ -155,7 +155,7 @@ def create_claim_to_ix(self, owner: SolPubKey,
             to_acct.address,
             amount
         ).build_transaction(
-            {'nonce': nonce, 'gasPrice': 0}
+            {'nonce': nonce, 'gasPrice': 0, 'from': signer_acct.address}
```

Comment:
Ethereum transaction doesn't have 'from'-property.
It is calculated from the signature.
The signature is formed from the private key.
This modification is surplus.

Reply:
This change for eth_estimateGas. Without specifying the `from` field, the proxy simulates a transaction with a default `0x000...000` sender and fails due to missing authority. This authority is created from the `contract` and `from` fields.